Step07/app/build.gradle

```diff
@@ -16,5 +16,11 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
 	runtimeOnly 'org.springframework.boot:spring-boot-devtools'
+	compileOnly 'org.projectlombok:lombok:1.18.12'
```

Comment:
Because this code is a workshop, and because not everyone is familiar with lombok, I wanted to use vanilla java code and not carry extra dependencies doing _black magic_. I admit though that lombok is a good library.

Reply:
I used it when I started to modify the code, but it has been removed since.